netmiko/cisco/cisco_vxr_ssh.py

```diff
@@ -0,0 +1,634 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+import re
+import time
+import logging
+import os
+import paramiko
+import warnings
+import socket
+
+from netmiko.cisco.cisco_xr import CiscoXrSSH
+from netmiko.netmiko_globals import BACKSPACE_CHAR
+from netmiko.utilities import get_structured_data
+from netmiko.cafy_custom_exceptions import SessionDownException, PromptNotFoundException, PatternNotFoundException
+from netmiko.cafy_custom_exceptions import ConfigCommitError, ConfigModeEnterError, ConfigModeExitError, LOOP_DELAY
+
+work_dir = os.getenv('CAFYKIT_WORK_DIR')
+if work_dir:
+    formatter = logging.Formatter(fmt='%(asctime)s %(levelname)-8s %(message)s',
+                                  datefmt='%Y-%m-%d %H:%M:%S')
+    paramiko_log_folder = os.path.join(work_dir, 'test_paramiko.log')
+    paramiko.util.log_to_file(paramiko_log_folder, level="DEBUG")
+    fh = logging.FileHandler(paramiko_log_folder)
+    fh.setFormatter(formatter)
+    log = logging.getLogger("netmiko")
+    log.addHandler(fh)
+    log.setLevel(logging.DEBUG)
+    log.addHandler(fh)
```

Comment:
remove this duplicate addition of handler. also we need to add a check if handler is already added. this is to avoid paramiko logs printed twice in current releases

Reply:
removed.